staging/src/github.com/kubeedge/beehive/pkg/core/module_simple.go

```diff
@@ -0,0 +1,82 @@
+package core
+
+import klog "k8s.io/klog/v2"
+
+// SimpleModule is the basic structure for rapid development of a module.
```

Comment:
If it will be better name the struct and file with `BasicModule` `basic_module.go`?

Reply:
I also thought about using "basic", but this is used directly, not an abstract struct.